src/Components/Assets/AssetServiceEditModal.tsx

```diff
@@ -169,19 +168,32 @@ export const AssetServiceEditModal = (props: {
               className="col-span-6 sm:col-span-3"
               data-testid="asset-last-serviced-on-input"
             >
-              <FieldLabel>Serviced On</FieldLabel>
-              <DateInputV2
+              <DateFormField
+                // eslint-disable-next-line i18next/no-literal-string
+                label="Serviced On"
```

Comment:
You should add translations and use it here instead of suppressing the eslint warning.